dictionaries/condensed-strokes.json

```diff
@@ -1,6 +1,10 @@
 {
+"TRA*FL/HRER": "traveller",
+"PWEURT/-PBS": "bitterness",
+"EUPL/PHORT/AL": "immortal",
```

Comment:
As per https://github.com/didoesdigital/steno-dictionaries/issues/288#issuecomment-609504922, I would not add this entry using the phonetic `/AL` but instead stick to the entry using a prefix and suffix stroke: `"EUPL/PHORT/A*L": "immortal",`

Reply:
Yep, understood. Fixed.